code/game/objects/items/storage/boxes.dm

```diff
@@ -912,7 +912,7 @@
 
 /obj/item/storage/box/rubbershot
 	name = "box of rubber shots"
-	desc = "A box full of rubber shots, designed for riot shotguns."
+	desc = "A standard box full of rubber shots, designed for riot shotguns. The box is manufactured by Nano Manufacturing."
```

Comment:
The added bits at the end should be removed, these are used generically by all factions and can be crafted in your hands. As per our official lore nano weapons isn't in a place to exist.